notesserver/highlight.py

```diff
@@ -39,13 +39,28 @@ def build_search_kwargs(self, *args, **kwargs):
             highlight_options = {
                 'fields': {
                     content_field: {'store': 'yes'},
-                }
+                    'tags': {'store': 'yes'},
+                },
             }
             if isinstance(highlight, dict):
                 highlight_options.update(highlight)
             res['highlight'] = highlight_options
         return res
 
+    def _process_results(
```

Comment:
@jimabramson I was struggling with the correct way to return the highlighted tags from haystack. It looks like haystack can only return the highlight from the content field (`text` in this case) using the ElasticsearchSearchBackend: https://github.com/django-haystack/django-haystack/blob/be67c37e3d23eed96d675e0c56e85eb6de0d484b/haystack/backends/elasticsearch_backend.py#L614. However, we need the highlight information for both the text and the tags. The code below is a bit of a hack, and I'm open to cleaner ways of returning this information.